MCDataProducts/inc/PhysicalVolumeInfoMultiCollection.hh

```diff
@@ -12,6 +12,7 @@
 
```

Comment:
Why do we need this file (or PhysicalVolumeInfoMultiCollection.cc)? Why not move its content to PhysicalVolumeInfo.hh/.cc, like we do with the Collection etc. typedefs in other data products.

Reply:
The header exists since 2013, per the original Mu2e convention of having a header file with a matching name for each data type.  I am not arguing this is the best way to go, but I do not recall any discussions/decisions on changing that.   Collecting several related types into a single header would require the compiler to open fewer files, which is good.    Once we have an official guidance on that issue we can consolidate header files for this and other data types.  But this restructuring is not related to the current PR.    But the way,  this header is referenced in  
   ack -l PhysicalVolumeInfoMultiCollection.hh|wc -l
   17
source files.

Reply:
The discussion and decision to consolidate typedefs into the primary data product header file did take place, but it was never documented as a formal policy.  We have taken a target-of-opportunity approach to dealing with it, but it can be skipped for this PR.